### ISSUE TITLE
Why?

### ISSUE THREAD

**Issue by AdamxzV3** [Euronymou5/Roblox-Phish#1]
Why is it encoded in base64? are you scared of something, decoded the start from base64 and it seems clean not check whole thing yet.

**Comment by Euronymou5** [comment]
scared why would I be scared?, the reason why the code is obfuscated is because of the code thieves, I can pass the source code without problems.

**Comment by AdamxzV3** [comment]
Good point tbh, skids are annoying

**Comment by spk-ie** [comment]
tbh the obfuscation method isnt that great but still its good to have one

**Comment by Euronymou5** [comment]
> tbh the obfuscation method isnt that great but still its good to have one

yes, I will probably upload the source code, I never put much effort into looking for a good obfuscator because I hardly need them lol

**Comment by beqy123** [comment]
newbie here just interested how this works

**Comment by beqy123** [comment]
can we have a source code

**Comment by Euronymou5** [comment]
> can we have a source code

Yes, as soon as I can I will upload the source code of the program.
